docs/MergingPrs.md

```diff
@@ -0,0 +1,34 @@
+# Merging a PR
+
+**Who can merge?** 
+A PR is to be merged by the author of the PR if the author has commit privileges. 
+Else it should be merged by the main reviewer.  
```

Comment:
@damithc Since all we do is press the "merge" button, why can't the project manager do this directly? This would shorten turnaround time.